### PR TITLE
A new index page from Tianfeng Lian

### PR DESCRIPTION
**Pull request by liantianfeng**
A index page made from the original design.

### CODE REVIEW

**Comment by liantianfeng** [comment]
index page from Tianfeng Lian

File: templates/boards/welcome.html [hunk 1]
```diff
@@ -2,5 +2,32 @@
...
+        </p>
+        <p><a href="#" target="_blank" class="btn btn-success">Learn More &raquo;</a></p>
+      </div>
+    </div>
+    <div class="col-12 col-md-6">
+      <img src="{% static 'img/front.jpg' %}" class="mt-3 img-fluid" alt="Responsive image">
+    </div>
+  </div>
+</div>
+{% endblock %}LoginLogin
```
Comment (jitingcn) [inline templates/boards/welcome.html]: Maybe you can replace the picture with a signup button and add a full-screen background picture?

File: templates/boards/welcome.html [hunk 2]
```diff
@@ -2,5 +2,32 @@
...
+      </div>
+    </div>
+    <div class="col-6 col-md-3">
+      <div class="p-3">
+        <h2>Updates</h2>
+        <p>sometext sometext sometext sometext sometext sometext sometext sometext sometext sometext sometext sometext
+        </p>
+        <p><a href="#" target="_blank" class="btn btn-success">Learn More &raquo;</a></p>
+      </div>
+    </div>
```
Comment (jitingcn) [inline templates/boards/welcome.html]: Try to make the those text bigger.

**Review by RogueTea — DISMISSED** [review]
I agree with the requested changes that jitingcn suggested

**Review by jitingcn — APPROVED** [review]

**Pull request merged**